server/services/forms/schemas/contribution-request.js

```diff
@@ -51,25 +51,25 @@ module.exports = async options => {
         )
         .withField(
             Component('date', 'contributionRequestDate')
-                .withValidator('required', 'Contribution request date is required')
-                .withValidator('isValidDate', 'Contribution request date must be a valid date')
-                .withValidator('isBeforeToday', 'Contribution request date must be in the past')
+                .withValidator('required')
+                .withValidator('isValidDate')
+                .withValidator('isBeforeToday')
                 .withProp('label', 'Contribution request date')
                 .withProp('disabled', isReadOnly)
                 .build()
         )
         .withField(
             Component('date', 'contributionDueDate')
-                .withValidator('required', 'Contribution due date is required')
-                .withValidator('isValidDate', 'Contribution due date must be a valid date')
-                .withValidator('isAfterToday', 'Contribution due date must not be in the past')
+                .withValidator('required')
+                .withValidator('isValidDate')
+                .withValidator('isAfterToday')
                 .withProp('label', 'Contribution due date')
                 .withProp('disabled', isReadOnly)
                 .build()
         )
         .withField(
             Component('text-area', 'contributionRequestNote')
-                .withValidator('required', 'What you are requesting is required')
+                .withValidator('required')
```

Comment:
I think this'll mean it ends up with "Contribution Request Notes is required", which reads strangely.

Reply:
Validation is based on the label of the component. In this case as we set the label to be `What you are requesting` on line 73. This will read as 'What you are requesting is required' which is what it said before.